gramex/handlers/mlhandler.py

```diff
@@ -43,23 +49,31 @@
     'pipeline': True,
     'nums': [],
     'cats': [],
-    'target_col': None
+    'target_col': None,
+    'index_col': None
 }
 ACTIONS = ['predict', 'score', 'append', 'train', 'retrain']
 DEFAULT_TEMPLATE = op.join(op.dirname(__file__), '..', 'apps', 'mlhandler', 'template.html')
-search_modelclass = lambda x: locate(x, MLCLASS_MODULES)  # NOQA: E731
+is_statsmodel = lambda x: x.__module__ == 'gramex.sm_api'  # NOQA: E731
+
+
+def search_modelclass(mclass):
+    _class = locate(mclass, MLCLASS_MODULES)
+    if _class is None:
+        raise ImportError(f'{mclass} not found.')
+    return _class
 
 
-def _fit(model, x, y=None, path=None, name=None):
+def _fit(model, x, y=None, path=None, name=None, **kwargs):
     app_log.info('Starting training...')
     try:
-        getattr(model, 'partial_fit', model.fit)(x, y)
+        result = getattr(model, 'partial_fit', model.fit)(x, y, **kwargs)
         app_log.info('Done training...')
         joblib.dump(model, path)
         app_log.info(f'{name}: Model saved at {path}.')
+        return result
```

Comment:
Ensure `_fit()` always consistently returns something.

This may be a good time to (incrementally) introduce typing.